docs/keycloak_deployment.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Deploy Keycloak
+nav_order: 3000
+---
+
+# Keycloak Deployment Guide
+
+Keycloak is the service we use for managing users and login sessions. Here we'll cover how to setup a fresh keycloak service from scratch
+
+## Get access to the STAR AWS account
+
+Contact Arend Peter ``@Arend Peter`` on slack to get a AWS user on the STAR AWS account
+
+## Run Instance
+
+1. Go to the EC2 Console
+1. Select "Launch Instance"
+1. Configure as follows
+    * Name: Keycloak Server
+    * OS: Amazon Linux 2 64 Bit (I used ami-0cff7528ff583bf9a but you can use the latest)
+    * Instance Type: t2.micro (Keycloak needs at least .5 GB of memory and this is the cheapest one with 1 GB of memory)
+    * Key Pair: Create Key pair (RSA/pem), and download the pem file (keep the pem file in a safe you'll need it later to access the instance)
+    * Network Settings: Select "Allow HTTPs traffic from the internet"
+    * Configure Storage: Select advanced, 16 GB (default is 8 GB, that might be enough but it just seemed low to me), and set Encrypted to Yes, and KMS Key to aws/ebs
```

Comment:
do we know what we need to store?  Theoretically just the keycloak code + everyone's emails and passwords?

Reply:
Yep, that's all for the moment, although I want to look into linking keycloak to a separate sql database